### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by melanierogan**
Added a toggle to filter by Emotion or Action categorised posts, we also added a drop-down option of choosing Emotion or Action category when posting a new post. 

Added login page and styling, styled signup page and sorted out user flow with sessions.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+      </div>
+
 
-<div class="card fluid p-4">
 <% @posts.each do |p| %>
-  <%= p.choice %><br>
-  <%= simple_format(p.body) %><br>
-  <%= p.address %><br>
-  <%= User.find(p.user_id).username %><br>
+<div id="bum" class="card fluid p-4 <%= p.choice %>">
```
Comment (JaydaKad) [inline app/views/posts/index.html.erb]: do we want a bum id?

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,11 +1,27 @@
-<p>posts is saved</p>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script type="text/javascript">
+
+    $(document).ready(function() {
+      $('#bum_toggle').change(function (){
```
Comment (JaydaKad) [inline app/views/posts/index.html.erb]: do we want a bum toggle?

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (melanierogan) [reply app/views/posts/index.html.erb]: No more bums! 🍑

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (melanierogan) [reply app/views/posts/index.html.erb]: renaming it `mol_toggle` for mel and ollie

**Pull request merged**